src/vde_backend/retrieveDB.py

```diff
@@ -0,0 +1,221 @@
+__title__ = "retrieveDB"
+__version__ = "1.0.0"
+__author__ = "Alexandre Heneffe"
+__license__ = "MIT"
+
+
+""" 
+Script to save Flukso electrical data from the Cassandra database
+"""
+
+
+# standard library
+from datetime import timedelta
+import os
+import os.path
+import argparse
+
+# 3rd party packages
+import pandas as pd
+
+# local sources
+from constants import (
+	CASSANDRA_KEYSPACE, 
+	TBL_POWER
+)
+import pyToCassandra as ptc
+
+from utils import (
+	getDatesBetween, 
+	getLastRegisteredConfig
+)
+
+DATA_LOCAL_PATH = 'output/retrieved_data/' # TODO : put it in constants.py
```

Comment:
The TODO seems quick to achieve. Why not doing it directly?

Reply:
This constant no longer exists. It is replaced by the argument "output_filename".